BetterBioReactor/Patchers/BaseBioReactor_Patchers.cs

```diff
@@ -78,7 +78,7 @@ internal static bool Prefix(BaseBioReactor __instance, ProtobufSerializer serial
     }
 
     [HarmonyPatch(typeof(BaseBioReactor))]
-    [HarmonyPatch(nameof(BaseBioReactor.OnProtoDeserialize))]
+    [HarmonyPatch(nameof(BaseBioReactor.OnProtoDeserializeAsync))]
```

Comment:
Does this not need to be in an if else between games?

Reply:
wait, are you shitting me? There's an OnProtoDeserializeAsync in BelowZero? Maybe that's why I had such a time with DWEB...